Contributing.md

```diff
@@ -0,0 +1,96 @@
+
+You can see the current teams and working groups in the ["Teams and Working Groups"](https://github.com/nodejs/community-committee#current-teams-and-working-groups) section of the README.md file - if there is one that seems to cover an area of particular interest to you, you should take a look at the Team or Working Group's repo to see how you can get further involved.
+
+# Logging Issues
+
+Log an issue for any question or problem you might have. When in doubt, log an issue,
+any additional policies about what to include will be provided in the responses. The only
+exception is security disclosures which should be sent privately. If you have a concern that is Code
+of Conduct related and do not feel comfortable reporting publicly, please reach out to an individual listed under Community Committee Collaborators
+
+Collaborators may direct you to another repository, ask for additional clarifications, and
+add appropriate metadata before the issue is addressed.
+
+Please be courteous, respectful, and every participant is expected to follow the
+project's Code of Conduct.
+
+# Contributions
+
+Any change to resources in this repository must be through pull requests. This applies to all changes
+to documentation, code, binary files, etc. Even long term collaborators and WG members must use
+pull requests.
+
+No pull request can be merged without being reviewed.
+
+For non-trivial contributions, pull requests should sit for at least 36 hours to ensure that
+contributors in other timezones have time to review. Consideration should also be given to
+weekends and other holiday periods to ensure active collaborators all have reasonable time to
+become involved in the discussion and review process if they wish.
+
+The default for each contribution is that it is accepted once no collaborator has an objection.
+During review collaborators may also request that a specific contributor who is most versed in a
+particular area gives a "LGTM" before the PR can be merged. There is no additional "sign off"
+process for contributions to land. Once all issues brought by collaborators are addressed it can
+be landed by any collaborator.
+
+In the case of an objection being raised in a pull request by another collaborator, all involved
+collaborators should seek to arrive at a consensus by way of addressing concerns being expressed
+by discussion, compromise on the proposed change, or withdrawal of the proposed change.
+
+If a contribution is controversial and collaborators cannot agree about how to get it to land
+or if it should land then it should be escalated to the WG. WG members should regularly
+discuss pending contributions in order to find a resolution. It is expected that only a
+small minority of issues be brought to the WG for resolution and that discussion and
+compromise among collaborators be the default resolution mechanism.
+
+# Becoming a Member and Collaborator
+
```

Comment:
This should be updated to talk about how to become a member of the user-feedback team